COMMUNITY_BADGES.md

```diff
@@ -0,0 +1,67 @@
+# OPA Community Badges
+
+Hello OPA Community Members, we thank you for taking the time to be a part of the community and for all of the work you’ve put in. As thanks, the OPA community now offers badges to recognize your contributions to the community.
+
+As of right now, these badges are simple tokens of appreciation that we will proudly display in the OPA GitHub repository. As the program develops, we will provide digital badges and certificates for members to display on their Linkedin and Twitter accounts.
+
+## Contributor Badge
+
+We appreciate contributions to the OPA project, no matter how big or small. So if you’ve contributed a dozen lines of code or corrected a typo in the docs, let us know!
+
+### Requirements
+
+- 1 Merged PR
+
+## Spokesperson Badge 
+
+For community members that like to create OPA content, share your work to receive the OPA Spokesperson Badge.
+
+### Requirements
+
+One or more pieces of publically posted content such as any of the following (Available for non-vendor related content only):
+
+- Conference Talk
+- Meetup presentation
+- Online Video Recording
+- Technical blog or OPA tutorial 
+
+## Super User Badge 
+
+This badge is for members that know the value of running OPA in production. Add your organization to the ADOPTERS.md file, then share a quick snippet about how OPA is used in your production environment to earn this badge. 
+  
+### Requirements
+
+- Deploy or Support OPA in a production environment
+- Add your organization to the ADOPTERS.md file
+
+### Benefits
+
+- Access to a private OPA Super Users Slack Channel
+- Support writing an OPA CFP to grow your portfolio
```

Comment:
What kind of support are we talking about here ? For ex, reviewing CFPs by OPA maintainers ?

Reply:
Anyone on the OPA team is fine, the goal here is to get more organizations to share their story.